crypto/crypto.ts

```diff
@@ -215,6 +218,11 @@ const stdCrypto: StdCrypto = ((x) => x)({
         // and the data is a single buffer,
         isBufferSource(data)
       ) {
+        if (length !== undefined) {
+          throw new TypeError(
```

Comment:
The native `crypto.subtle.digest` seems ignoring `length` for the case where `length` is irrelevant. Throwing here feels inconsistent with it.

Reply:
Good point. I've reverted and instead added a note saying that `length` will be ignored in some cases.